### PR TITLE
Document `pull_from_upstream`

### LINKED ISSUE #495: Document pull-from-upstream job
Document the job implemented in packit/packit-service#1578.

*part of packit/packit-service#1576 epic*

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #495

### CODE REVIEW

File: content/docs/Configuration/_index.md [hunk 1]
```diff
@@ -722,6 +722,45 @@ jobs:
...
 This config would update Fedora Rawhide and Fedora 35 dist-git branches.
 
+##### pull_from_upstream
+
+A dist-git only job that opens a new dist-git pull request in
+src.fedoraproject.org when a new upstream release happens using a notification
+from [release-monitoring.org](https://release-monitoring.org/).
+
+This job utilizes the same logic as `propose_downstream` with the only
+exception it is defined and executed in dist-git.
```
Comment (jpopelka) [inline content/docs/Configuration/_index.md:732]:
```suggestion
exception that it is defined and executed in dist-git.
```

File: content/docs/Configuration/_index.md [hunk 2]
```diff
@@ -722,6 +722,45 @@ jobs:
...
+
+Use [`issue_repository`](#issue_repository) option to get information about
+possible failures during the update process.
+
+Requirements:
+* The job is defined in dist-git.
```
Comment (jpopelka) [inline content/docs/Configuration/_index.md:738]: What if I had it configured differently (`dist_git_branches`) for each branch? Which config/branch would be the source of truth then?

File: content/docs/Configuration/_index.md (hunk 1 above)
Comment (TomasTomecek) [reply content/docs/Configuration/_index.md:732]: you're right: either comma is missing or "that", thanks!

File: content/docs/Configuration/_index.md (hunk 2 above)
Comment (TomasTomecek) [reply content/docs/Configuration/_index.md:738]: @lbarcziova could you please answer and suggest how to capture this use case?

File: content/docs/Configuration/_index.md [hunk 3]
```diff
@@ -722,6 +722,45 @@ jobs:
...
+from [release-monitoring.org](https://release-monitoring.org/).
+
+This job utilizes the same logic as `propose_downstream` with the only
+exception that it is defined and executed in dist-git.
+
+Use [`issue_repository`](#issue_repository) option to get information about
+possible failures during the update process.
+
+Requirements:
+* The job is defined in dist-git.
```
Comment (lbarcziova) [inline content/docs/Configuration/_index.md:738]:
```suggestion
* The job is defined in Packit config in the default branch of the dist-git repository.
```

File: content/docs/Configuration/_index.md [hunk 4]
```diff
@@ -722,6 +722,45 @@ jobs:
...
+* Upstream release monitoring is active for the package.
```
Comment (lbarcziova) [inline content/docs/Configuration/_index.md:739]:
```suggestion
* Upstream release monitoring is active for the package ([see](https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring/), the monitoring status in dist-git should be set to `Monitoring`). 
```

File: content/docs/Configuration/_index.md (hunk 2 above)
Comment (lbarcziova) [reply content/docs/Configuration/_index.md:738]: We take the config from the default branch of the dist-git repository. I think for this kind of use case user could configure multiple jobs :thinking:

**Comment by TomasTomecek** [comment]
@jpopelka @lbarcziova rereview please, added a bit more clarification and rewording.

**Review by lbarcziova — APPROVED** [review]
thanks!

**Review by jpopelka — APPROVED** [review]

**Pull request merged**